Cargo.toml

```diff
@@ -23,6 +23,7 @@ serde_json = "1.0"
 log = "0.4"
 env_logger = "0.9"
 tokio = { version = "1.9", features = ["macros", "rt-multi-thread"] }
+version-compare = "0.0.11"
```

Comment:
In current [`Cargo.lock`](https://github.com/nervosnetwork/ckb-indexer/blob/master/Cargo.lock#L1718-L1722), there already has a crate for version comparison: [version_check](https://crates.io/crates/version_check), and it's more popular.

How about replace `version-compare` with `version_check`?

Reply:
`version_check` is used to parse rustc version, ckb version string is not compatible with rustc version string.

this line will return none:
```
Version::parse("0.43.1 (15427e0 2021-07-16)")
```